transcom/mymove
# B 18188 int removed deprecated personally procured moves model and devseed data

B-18448

Summary
Removed PersonallyProcuredMoves model from MyMove Due to the extensive modifications to the tests and active code needed to fully remove references to this model, this component will require extensive testing

##Testing Steps

Verify all related Unit tests pass (Changes were all Go, but could indirectly impact JS).
Verify the ability to create PPMs as a customer
Approve/Reject/modify it as a Service Counselor
When signing the shipment as Customer verify that all fields are being populated, and no errors occur
